tests/integration/backward_compatible/db_test.py

```diff
@@ -0,0 +1,123 @@
+from typing import List
+
+from hazelcast import HazelcastClient
+from hazelcast.config import Config
+from hazelcast.db import connect, Connection
+from .sql_test import (
+    SqlTestBase,
+    compare_server_version_with_rc,
+    compare_client_version,
+    SERVER_CONFIG,
+    JET_ENABLED_CONFIG,
+    Student,
+)
+
+
+class DbapiTestBase(SqlTestBase):
```

Comment:
The test should be guarded against the client and server version

Reply:
Moved these tests out out of backward compatibility.